plugins/optimization-detective/optimization.php

```diff
@@ -118,12 +118,6 @@ function od_can_optimize_response(): bool {
 		is_customize_preview() ||
 		// Since the images detected in the response body of a POST request cannot, by definition, be cached.
 		( isset( $_SERVER['REQUEST_METHOD'] ) && 'GET' !== $_SERVER['REQUEST_METHOD'] ) ||
-		// The aim is to optimize pages for the majority of site visitors, not for those who administer the site, unless
-		// in 'plugin' development mode. For admin users, additional elements will be present, like the script from
-		// wp_customize_support_script(), which will interfere with the XPath indices. Note that
-		// od_get_normalized_query_vars() is varied by is_user_logged_in(), so membership sites and e-commerce sites
-		// will still be able to be optimized for their normal visitors.
-		( current_user_can( 'customize' ) && ! wp_is_development_mode( 'plugin' ) ) ||
```

Comment:
What about the development mode condition here? That seems unrelated to what this PR is doing, so shouldn't it be kept?

Reply:
The development mode condition was added to force optimization to be performed even when the user is an administrator. However, now optimization applies by default for everyone including administrators, so this is obsolete).

Reply:
For context, see #1423 fixed via #1700.